example-transports.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+const pino = require('./')
+const { tmpdir } = require('os')
+const { join } = require('path')
+
+const file = join(tmpdir(), `pino-${process.pid}-example`)
+
+const transport = pino.transport([{
+  level: 'warn',
```

Comment:
can we support

```
levels: ['warn', 'error', 'fatal']
```

I think this could be fairly easily achieved - we would just map the levels array to 

```
[{ level: 'warn', ...otherStuff }, { level: 'error', ...otherStuff }, {level: 'fatal, ...otherStuff }]
```